core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/blueprint/BlueprintV4RequestToBlueprintConverter.java

```diff
@@ -78,6 +78,7 @@ public Blueprint convert(BlueprintV4Request json) {
                 blueprint.setStackName(blueprintUtils.getCDHDisplayName(blueprintJson));
                 blueprint.setHostGroupCount(blueprintUtils.countHostTemplates(blueprintJson));
                 blueprint.setStackVersion(blueprintUtils.getCDHStackVersion(blueprintJson));
+                blueprint.setBlueprintUpgradeOption(blueprintUtils.getBlueprintUpgradeOptionForGA());
```

Comment:
Since we have started to set BlueprintUpgradeOption as part of this change, it should be null for existing records. Don't have any requirement to populate this column for existing records?

Reply:
there is no requirement to populate for all existing record Adarsh. UpgradePreConditionService: line 62 will take care of it